src/components/AppBars/AppBarBase/__test__/AppBarBase.test.tsx

```diff
@@ -12,8 +13,14 @@ describe("<AppBarBase />", () => {
         return <AppBarBase {...props} />;
     };
 
+    const TestComponent = (props: AppBarBaseProps) => (
+        <TestThemeProvider>
+            <AppBarBaseTest {...props} />
+        </TestThemeProvider>
+    );
```

Comment:
Is this what you mean? ![image](https://user-images.githubusercontent.com/26225039/140064780-78ba33cd-5122-4de1-a9de-d76fc0f1504e.png)

Reply:
How can it get the correct theme if useTheme is outside of the theme provider?

Reply:
yes, that's what I mean.
For the second question: I thinh that because all theme attributes which is used to test in this file are not customized by our relay theme.